### PR TITLE
Dotnet/wire up feature and add docs

### PR DESCRIPTION
**Pull request by Kpopovs1**

### CODE REVIEW

File: script-library/docs/dotnet.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# Azure CLI Install Script
```
Comment (bderusha) [inline script-library/docs/dotnet.md]:
```suggestion
# Dotnet CLI Install Script
```

File: script-library/docs/dotnet.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+```text
+./dotnet-debian.sh [dotnet version] [dotnet runtime only]
+```
+
+Or as a feature:
+
+```json
+"features": {
+    "dotnet": "latest"
```
Comment (bderusha) [inline script-library/docs/dotnet.md]:
```suggestion
    "dotnet": {
      "version": "latest",
      "runtimeOnly": false
    }
```

File: script-library/docs/dotnet.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+## Usage
+
+### Feature use
+
+To install these capabilities in your primary dev container, reference it in `devcontainer.json` as follows:
+
+```json
+"features": {
+    "dotnet": "latest"
```
Comment (bderusha) [inline script-library/docs/dotnet.md]:
```suggestion
    "dotnet": {
      "version": "latest",
      "runtimeOnly": false
    }
```

**Review by bderusha — CHANGES_REQUESTED** [review]
Overall this looks great!  A few quick comments and then we can merge it in

File: script-library/docs/dotnet.md (hunk 1 above)
Comment (Kpopovs1) [reply script-library/docs/dotnet.md]: Ahh thank you for catching this!

**Review by bderusha — APPROVED** [review]
🎸

**Pull request merged**